interfaces/builtin/kernel_module_control.go

```diff
@@ -29,9 +29,13 @@ const kernelModuleControlConnectedPlugAppArmor = `
   capability sys_module,
   @{PROC}/modules r,
 
-  # NOTE: needed by lscpu. In the future this may be moved to system-trace or
-  # system-observe.
+  # FIXME: moved to physical-memory-observe (remove this in series 18)
```

Comment:
Just as an observation. When we do series 18 we will allow people to co-install 16 and 18 base snaps and apps can use either of those. It just struck me that we should probably teach the interface layer the base snap the rule is for. The `kernel-module-control` interface will have to keep supplying access to `/dev/mem` even when 18 is around if the interface is connected to a snap using `base-ubuntu-16`.